docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>Show the plot output and add that to the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>